.gitignore

```diff
@@ -137,7 +137,6 @@ dmypy.json
 .pyre/
 
 # isort config is in pyproject.toml
-.isort.cfg
```

Comment:
there should be no `isort.cfg`, since all in pyporject.toml

this was to prevent anyone from pushing one...